### DISCUSSION TITLE
POST https://vaccine.kakao.com/api/v2/reservation VM25:471 403

### DISCUSSION THREAD

**Discussion by hakdlf** [kimytsc/covid-rest-vaccine-macro#33] [Q&A] [answered]
<img width="411" alt="스크린샷 2021-08-10 오전 10 59 17" src="https://user-images.githubusercontent.com/87300969/128797008-f79e321b-2b69-4914-bf9e-6250aedc88f5.png">

사진과 같이
VM25:471 POST https://vaccine.kakao.com/api/v2/reservation 403
라고 뜨는데 무슨 문제일까요

**Comment by kimytsc** [comment, marked as answer]
/api/v2/reservation 403 로그는 예약 시도 실패시 나타나는 로그입니다.

카카오는 예약 시도 실패시 403 상태코드로 응답을 주는데, 이 과정에서 무조건적으로 `Failed to load resource: the server responded with a status of 403 ()`이 로그에 노출되게 됩니다.

화면에서 병원명과 실패사유가 변경된다면, 정상적입니다.

**Reply by hakdlf** [reply]
<img width="1440" alt="스크린샷 2021-08-10 오전 11 02 15" src="https://user-images.githubusercontent.com/87300969/128797241-6684e900-7a3e-468a-b014-73de47016adc.png">

다음과 같은 창에서 시간만 변한다면 정상적으로 진행되는 것으로 판단해도 되는 것일까요?

**Reply by kimytsc** [reply]
검은색으로 표시되는 시간이 갱신되고 있다면 리스트를 잘 가져오고 있는 상태입니다.

예약시도 내역에도 변경사항이 있는걸 보면, 정상으로 판단됩니다.
